<launch>

    <!-- main launch node -->
    <node pkg="pointgrey_ladybug" type="ladybug_camera_raw_display" name="ladybug_camera_display" output="screen">
    <!--<node pkg="pointgrey_ladybug" type="ladybug_camera" name="ladybug_camera" output="screen" launch-prefix="gdb -ex run &#45;&#45;args">-->

        <!-- camera properties -->

        <param name="framerate"               type="double" value="6.5"/> <!-- max 6.5 -->
        <param name="use_auto_framerate"      type="bool"   value="true"/>
        <param name="shutter_time"            type="double" value="0.05"/>
        <param name="use_auto_shutter_time"   type="bool"   value="true"/>
        <param name="gain_amount"             type="double" value="20"/>
        <param name="use_auto_gain"           type="bool"   value="true"/>

        <!-- post-processing -->
        <param name="jpeg_percent"            type="int"    value="80"/>
        <param name="scale"                   type="double" value="99"/>


    </node>


</launch>
